src/ucs/debug/debug.h

```diff
@@ -24,14 +19,11 @@ BEGIN_C_DECLS
  */
 void ucs_debug_disable_signal(int signum);
 
+void ucs_debug_asan_validate_address(const char *ptr_name, void *address,
+                                     size_t size);
 
-#ifdef __SANITIZE_ADDRESS__
```

Comment:
can we keep __SANITIZE_ADDRESS__ check here, to avoid extra function call overhead when not using the address sanitizer?

Reply:
wouldn't that force us to add this check everywhere this function is being called from?

Reply:
just need to restore the deteled ifdef. it means the check affects the macro definition, and code using the macro does not need to change.